components/SidePillTag.vue

```diff
@@ -0,0 +1,268 @@
+<script setup lang="ts">
+import { ref, computed, onMounted } from 'vue'
+
+const STORAGE_KEY = 'sidepilltag-dismissed'
+const EXPIRATION_HOURS = 12
+
+const props = withDefaults(defineProps<{
+  title?: string
+  ctaText?: string
+  storeUrl?: string
+  icon?: string
+
+  top?: number
+  right?: number          
+  rightClosed?: number    
+  edgeNudge?: number      
+  height?: number
+  openWidth?: number
+  peekWidth?: number
+
+  lineColor?: string
+  lineWidth?: number
+  radius?: number
+
+  useElementPlus?: boolean
+}>(), {
+  // ---- conteúdo ----
+  title: 'Hemocione',
+  ctaText: 'INSTALAR APP',
+  storeUrl: 'https://link.hemocione.com.br/download-app',
+  icon: '/images/logo.svg',
+
+  // ---- defaults (os que você passava no app.vue) ----
+  top: 96,
+  right: 16,
+  rightClosed: 0,
+  edgeNudge: 0.5,
+  height: 100,
+  openWidth: 320,
+  peekWidth: 70,
+
+  lineColor: '#E54B4B',
+  lineWidth: 3,
+  radius: 48,
+
+  useElementPlus: false,
+})
+
+const open = ref(false)
+const visible = ref(true) // controla se deve renderizar (regra de 12h)
```

Comment:
```suggestion
const isVisibleSidePillTag = ref(true)
```

Sempre tenta passar a informação pelo código. Adiciona comentários quando realmente quiser dar mais contexto sobre algo; pra não perder o propósito 🙏🏼